src/toil/lib/docker.py

```diff
@@ -148,7 +151,18 @@ def _docker(job,
     job.defer(_fixPermissions, tool, workDir)
 
     # Make subprocess call
-    call = baseDockerCall + [tool] + parameters
+
+    # If parameters is list of lists, treat each list as separate command and chain with pipes
+    if len(parameters) > 0 and type(parameters[0]) is list:
+        # When piping, all arguments get passed as a single string to bash -c.
+        # We try to support spaces in paths by wrapping them all in quotes first.
+        quoted_params = []
+        for params in parameters:
+            quoted_params.append(['\'{}\''.format(p) for p in params])
```

Comment:
@glennhickey this might help you out https://docs.python.org/2/library/pipes.html#pipes.quote